code/game/machinery/telecomms/machine_interactions.dm

```diff
@@ -126,7 +126,7 @@
 						remove_link(T)
 					network = params["value"]
 					links = list()
-					log_game("[key_name(operator)] has changed the network for [src] at [AREACOORD(src)] to [network].")
+					operator.log_message("has changed the network for [src] at [AREACOORD(src)] to [network].", LOG_GAME)
```

Comment:
```suggestion
					operator.log_message("has changed the network for [src] to [network].", LOG_GAME)
```

Reply:
Just so this review isn't 100 lines long, you get the idea. I know you didn't add these, but just for housekeeping.